src/box/iproto.cc

```diff
@@ -2872,6 +2872,48 @@ iproto_session_notify(struct session *session,
 
 /** }}} */
 
+/**
+ * Sends a packet with the given header and body over the IPROTO session's
+ * socket.
+ * On success, a packet is written to the session's output buffer, which is
+ * flushed asynchronously using Kharon.
+ *
+ * @param sid IPROTO session identifier
+ * @param header MsgPack encoded header
+ * @param header_end MsgPack encoded header end
+ * @param body MsgPack encoded body
+ * @param body_end MsgPack encoded body end
+ * @retval -1 on error (check box_error_last())
+ * @retval 0 on success
+ *
+ * @exception ER_MEMORY_ISSUE failed to allocate memory for sending packet
```

Comment:
We don't throw exceptions. But either way, even just listing all possible errors usually makes very little sense. Such lists tend to outdate either from start or very soon, because nobody updates them really.